.github/ISSUE_TEMPLATE/incident-response.md

```diff
@@ -0,0 +1,59 @@
+---
+name: Incident response
+about: Use to track the steps that should happen during an incident, such as a data breach or unexpected downtime, even if it's just a suspicion.
+---
+
+_**Do not put sensitive information in this issue or Slack.** Use a file on Google Drive with access restricted._
+
+**Severity:** _High/Medium/Low_
+
+## Initiate
+
+1. [x] Create an issue from this template.
+1. [ ] Declare an incident in the relevant Slack channel, such as [#benefits-general][benefits-general].
+   - Include brief details about the concern.
+1. [ ] Start a video call.
+1. [ ] Share a link to the video call in Slack, asking relevant parties to join.
+1. [ ] Delegate subsequent tasks.
+
+## Assess
+
+- [ ] Determine the impact.
+- [ ] Assign the severity above:
+  - **High:** Possible/confirmed breach of sensitive information, such as production system credentials or personally-identifiable information (PII)
+  - **Medium:** Full Benefits downtime for more than 30 minutes
+  - **Low:** Partial service degredation
+
+### Medium/High incidents
+
+- [ ] Notify [#benefits-general][benefits-general].
+- [ ] If the incident is lasting more than 60 minutes or their customers' data has been breached, notify the impacted transit agencies.
+- [ ] If it's believed to be an ongoing attack (such as DDoS), notify DevSecOps.
+
+## Remediate
+
+- [ ] Take notes in the Slack thread.
```

Comment:
Maybe we could add something like ` / delegate someone for note-taking` to encourage doing so?